src/modules/clustering/hierarchical_clustering_method.cpp

```diff
@@ -25,13 +26,14 @@ std::vector<std::vector<Junction>> partition_junctions(std::vector<Junction> con
         {
             if (junction.get_mate1().seq_name != current_partition.back().get_mate1().seq_name ||
                 junction.get_mate1().orientation != current_partition.back().get_mate1().orientation ||
-                abs(junction.get_mate1().position - current_partition.back().get_mate1().position) > 50)
+                abs(junction.get_mate1().position - current_partition.back().get_mate1().position)
```

Comment:
doesn't this need `std::abs`? (I'm not sure, but if it works on every machine - fine!)
Update: at least below you use `std::`, so decide on one version please